From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Chris Bobbe <cbobbe@zulip.com>
Date: Mon, 25 Jul 2022 13:35:00 -0700
Subject: [shortcut] expo-web-browser: Translate a TS enum

Read more about our approach in the recent patch that did this for
expo-application.
---
 .../expo-web-browser/build/WebBrowser.js.flow |  3 +-
 .../build/WebBrowser.types.js.flow            | 35 +++++++++----------
 2 files changed, 19 insertions(+), 19 deletions(-)

diff --git types/expo-web-browser/build/WebBrowser.js.flow types/expo-web-browser/build/WebBrowser.js.flow
index 506ab86a1..969ce710d 100644
--- types/expo-web-browser/build/WebBrowser.js.flow
+++ types/expo-web-browser/build/WebBrowser.js.flow
@@ -12,11 +12,12 @@ import {
     type WebBrowserRedirectResult,
     type WebBrowserResult,
     WebBrowserResultType,
+    type WebBrowserResultTypeT,
     type WebBrowserWarmUpResult,
     type WebBrowserWindowFeatures,
 } from "./WebBrowser.types";
 
-export { WebBrowserAuthSessionResult, WebBrowserCompleteAuthSessionOptions, WebBrowserCompleteAuthSessionResult, WebBrowserCoolDownResult, WebBrowserCustomTabsResults, WebBrowserMayInitWithUrlResult, WebBrowserOpenOptions, WebBrowserRedirectResult, WebBrowserResult, WebBrowserResultType, WebBrowserWarmUpResult, WebBrowserWindowFeatures };
+export { WebBrowserAuthSessionResult, WebBrowserCompleteAuthSessionOptions, WebBrowserCompleteAuthSessionResult, WebBrowserCoolDownResult, WebBrowserCustomTabsResults, WebBrowserMayInitWithUrlResult, WebBrowserOpenOptions, WebBrowserRedirectResult, WebBrowserResult, WebBrowserResultType, WebBrowserResultTypeT, WebBrowserWarmUpResult, WebBrowserWindowFeatures };
 declare export function getCustomTabsSupportingBrowsersAsync(): Promise<WebBrowserCustomTabsResults>;
 declare export function warmUpAsync(browserPackage?: string): Promise<WebBrowserWarmUpResult>;
 declare export function mayInitWithUrlAsync(url: string, browserPackage?: string): Promise<WebBrowserMayInitWithUrlResult>;
diff --git types/expo-web-browser/build/WebBrowser.types.js.flow types/expo-web-browser/build/WebBrowser.types.js.flow
index 803676488..c4e0fdeb0 100644
--- types/expo-web-browser/build/WebBrowser.types.js.flow
+++ types/expo-web-browser/build/WebBrowser.types.js.flow
@@ -31,25 +31,24 @@ export type WebBrowserCustomTabsResults = {
     ...
 };
 
-/* tsflower-unimplemented: EnumDeclaration */
-/* 
-export declare enum WebBrowserResultType {
-    /**
-     * iOS only.
-     * /
-    CANCEL = "cancel",
-    /**
-     * iOS only.
-     * /
-    DISMISS = "dismiss",
-    /**
-     * Android only.
-     * /
-    OPENED = "opened",
-    LOCKED = "locked"
-} */
+declare export var WebBrowserResultType: {|
+  /**
+   * iOS only.
+   */
+  +CANCEL: 'cancel',
+  /**
+   * iOS only.
+   */
+  +DISMISS: 'dismiss',
+  /**
+   * Android only.
+   */
+  +OPENED: 'opened',
+  +LOCKED: 'locked',
+|}
+export type WebBrowserResultTypeT = $Values<typeof WebBrowserResultType>;
 
-export type WebBrowserResult = { type: WebBrowserResultType, ... };
+export type WebBrowserResult = { type: WebBrowserResultTypeT, ... };
 
 export type WebBrowserRedirectResult = {
     type: "success",
-- 
2.32.0

